pkg/trace/testutil/otlp.go

```diff
@@ -87,6 +98,17 @@ func setOTLPSpan(span ptrace.Span, s *OTLPSpan) {
 		insertAttributes(ev.Attributes(), e.Attributes)
 		ev.SetDroppedAttributesCount(e.Dropped)
 	}
+	links := span.Links()
+	for _, l := range s.Links {
+		li := links.AppendEmpty()
+		bytes, _ := hex.DecodeString(l.TraceID)
```

Comment:
I really liked how you've used short variables in the marshalling code. Why not do the same here? Use `b` or `buf`

Reply:
Done.